metadata-ingestion/tests/integration/remote/test_remote.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+from freezegun import freeze_time
+
+from datahub.ingestion.run.pipeline import Pipeline
+from tests.test_helpers import mce_helpers
+from tests.test_helpers.docker_helpers import wait_for_port
+
+FROZEN_TIME = "2021-08-24 09:00:00"
+
+
+@freeze_time(FROZEN_TIME)
+@pytest.mark.integration
+def test_remote_ingest(docker_compose_runner, pytestconfig, tmp_path, mock_time):
```

Comment:
Hey @xiphl, the `mock_time` fixture you're using here also mocks `time.time()` which is what generates audit stamps. I think this test will pass if you remove the fixture!

Reply:
yup that was the cause indeed!